src/applications/claims-status/components/ClaimCard/ClaimCardLink.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+
+export default function ClaimCardLink({
+  ariaLabel,
+  href,
+  text = 'View details',
+  onClick,
+}) {
+  return (
+    <Link
```

Comment:
Using a `Link` here instead of a `va-link` or `VaLink` because I want to avoid full refreshes when the links are clicked